package.json

```diff
@@ -26,11 +26,9 @@
     "add-contributor": "all-contributors add",
     "semantic-release": "semantic-release"
   },
-  "dependencies": {
-    "react": "^16.3.0"
-  },
+  "dependencies": {},
   "peerDependencies": {
-    "react": "^16.3.0"
+    "react": "16.x"
```

Comment:
Would you mind changing this back to `^16.3.0`? 16.x would allow less than 16.3.0, which would be incorrect.

Reply:
will fix 👍